rapid_pro/get_inbound_phone_numbers.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+
+from core_data_modules.logging import Logger
+from core_data_modules.util import TimeUtils
+from rapid_pro_tools.rapid_pro_client import RapidProClient
+from storage.google_cloud import google_cloud_utils
+from temba_client.v2 import Message
+
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Downloads all inbound messages from Rapid Pro instances and exports "
+                                                 "the phone numbers we heard from")
+
+    parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket")
+    parser.add_argument("rapid_pro_domain", help="URL of the Rapid Pro server to download data from")
+    parser.add_argument("rapid_pro_token_file_url", metavar="rapid-pro-token-file-url",
+                        help="GS URLs of a text file containing the authorisation token for the Rapid Pro server")
+    parser.add_argument("output_file_path", metavar="output-file-path",
```

Comment:
let export this as a CSV file with `URN:Tel` and `Name` column names so that we don't have to edit in excel/spreadsheet before uploading.

Reply:
Updated to export to CSV